mmv1/products/bigquery/Routine.yaml

```diff
@@ -73,6 +73,12 @@ examples:
       dataset_id: 'dataset_id'
       connection_id: 'connection_id'
       routine_id: 'routine_id'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'bigquery_routine_data_governance_type'
+    primary_resource_id: 'data_governance_type'
```

Comment:
Ah I renamed the resource in the test config since I thought "custom_masking_routine" would be more descriptive, so you'd need to update this name here, or revert that name in the test config